src/components/form/examples/custom-component-picker.tsx

```diff
@@ -81,6 +88,8 @@ export class CustomPickerExample implements FormComponent<number> {
     ) => {
         event.stopPropagation();
         this.change.emit(event.detail?.value);
+        // When the event below is emitted, the whole form is emptied 😲
+        this.limelChange.emit(event.detail?.value);
```

Comment:
I've tracked this down so far as to be reasonably sure this is because the event is not caught by the form, and it's therefore propagating past the form, and being caught by the handler listening for a `limelChange` coming from the form. Since that handler expects the whole formdata, the response when receiving a `null` because the value of this specific field is empty, is to empty all of the form-data.

However, I haven't quite figured out how to prevent that. I tried adding a listener for `limelChange`, but that didn't seem to do anything at all:

<img width="1069" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/ad7fe600-70ae-4d64-8ea6-7ba94d214858">

I'm thinking I probably need some help with this… @jgroth? 🙏